client/client-multi/client-objective-c/Kaa/persistance/KaaClientPropertiesState.m

```diff
@@ -474,9 +481,15 @@ - (KeyPair *)getOrGenerateKeyPair {
         return self.keyPair;
     }
     
-    DDLogDebug(@"%@ Generating new key pair", TAG);
-    self.keyPair = [KeyUtils generateKeyPair];
-    return self.keyPair;
+    if (!isAutogeneratedKeys) {
```

Comment:
IMHO, it's easier to read if-else statements if you swap the content like: if(true) rather then if(!true)
e.g. if (isAutogeneratedKeys) {..} else {..}